pymc/distributions/continuous.py

```diff
@@ -430,10 +433,13 @@ class HalfFlat(PositiveContinuous):
 
     rv_op = halfflat
 
+    def __new__(cls, *args, **kwargs):
+        kwargs["initval"] = kwargs.get("initval", "moment")
```

Comment:
```suggestion
        kwargs.setdefault("initval", "moment")
```

Same as for the `Flat` distribution